.github/workflows/build.yaml

```
@@ -31,6 +29,11 @@ jobs:
         run: |-
           cargo install cargo-make
```

Should we pull in the prebuilt binary here similar to what is done below?
https://github.com/knox-networks/ssi/blob/e2d20423de4ce5fa1f98c891cf6e604a69a964b9/.github/workflows/release_ffi.yaml#L75
```suggestion
        run: ./container/deps.sh --force cargo-make
```